### PR TITLE
Docs Refinement for GA

### PR DESCRIPTION
**Pull request by heyitswei**
Docs refinement

### CODE REVIEW

**Comment by jovezhong** [comment]
Why there are new files for confluent-cloud-source and apikey? They are already in main branch

**Comment by heyitswei** [comment]
Hi Jove, I think there are some duplicate files now

File: docs/create-api-key.md
Comment (jovezhong) [inline docs/create-api-key.md:1]: there is already a apikey.md

**Review by jovezhong — CHANGES_REQUESTED** [review]
Didn't use images correctly

**Comment by heyitswei** [comment]
Closing this PR as there are duplicate edits made - will send a new PR without duplicates

**Pull request closed without merging**